blog-content/tracing-distributed-systems-in-nextjs.md

```diff
@@ -0,0 +1,214 @@
+---
+title: "Tracing Distributed Systems in Next.js"
+createdAt: 2024-04-21T12:00:00Z
+readingTime: 6
+authorFirstName: Chris
+authorLastName: Esplin
+authorTitle: Software Engineer
+authorTwitter: 'https://twitter.com/chrisesplin'
+authorLinkedIn: 'https://www.linkedin.com/in/epsilon/'
+authorGithub: 'https://github.com/deltaepsilon'
+authorWebsite: 'https://www.chrisesplin.com/'
+authorPFP: 'https://firebasestorage.googleapis.com/v0/b/quiver-pixels-2020.appspot.com/o/F1EQ3eaBqkbEKEHBigolXIlmdut2%2F1408a808-60a6-4102-b636-08ab24041503.jpeg?alt=media&token=5f0ed5d8-c192-4aa3-a75b-3eb6cac9a552'
+tags: Tutorial
+metaTitle: "Tracing Distributed Systems in Next.js"
+---
+
+
+[![Watch on YouTube](/images/blog/tracing-distributed-systems-in-nextjs/distributed-tracing-livestream-16x9-play.png)](https://youtube.com/live/z9g-eOPwndk)
+
+## Distributed Tracing
+
+A naive approach to tracing creates a new trace for every service that is used to resolve a request.
```

Comment:
May just be github font, but had a tough time working out this sentence in my head. Once I read it a few times it makes sense, but wasn't sure if you noticed it. Possibly the use of `tracing` and `trace` so close together? Not blocking, but wanted to raise